thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/RawAnomalyResultDTO.java

```diff
@@ -22,8 +22,9 @@ public void setFunction(AnomalyFunctionDTO function) {
     this.function = function;
   }
 
+  // TODO: rename to getMetric and update the column name in DB?
   public String getMetric() {
-    return function.getMetric();
+    return function.getTopicMetric();
```

Comment:
just use get/set metric instead of "topicMetric" and have a javadoc explaining what metric is.